testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -958,7 +958,7 @@
 
 # Wait until a modal window with a specific content is shown
 When(/^I wait at most (\d+) seconds until I see modal containing "([^"]*)" text$/) do |timeout, title|
-  path = "//*[contains(@class, \"modal-body\") and contains(., \"#{title}\")]" \
+  path = "//*[contains(@class, \"modal-content\") and contains(., \"#{title}\")]" \
```

Comment:
Are we sure that this class changed for all our modal dialogs?

Reply:
No but we don't use this step anywhere else. For this it works like that